### DISCUSSION TITLE
seams with latest 2.5.5.x version

### DISCUSSION THREAD

**Discussion by pka4916** [supermerill/SuperSlicer#3166] [Q&A]
I did seams on the corners,  but  it seems to be all over the place..  it looks the same  as "nearest"

Is this a bug or am I missing something here?

**Comment by neophyl** [comment]
try it with the classic slicing engine.  I don't think arachne behaves properly with seams, at least the Prusa Slicer implementation was having troubles with them around the time that arachne was merged into SuSi so I'm guessing that prusa's fixes aren't in yet.

**Reply by pka4916** [reply]
Tried the classic as well, and it's also all over the place.
